src/bilder/images/docker_baseline_ami/deploy.py

```diff
@@ -85,16 +80,10 @@
 configure_vector(vector_config)
 vector_service(vector_config)
 
-for product in hashicorp_products:
-    configure_hashicorp_product(product)
```

Comment:
Curious if there's utility in keeping this here and just removing the step that sets up the services?

Reply:
I don't know. I would need to do a more methodical experiment with it. I'm a little too scattered at the moment. Intuitively, it feels like invoking once with one config object and then again later with a totally different config object is just asking for trouble but I don't have much to support that statement yet. I will make a note to look into it further.